tests/unit/test_session.py

```diff
@@ -81,7 +101,7 @@ class _Session(SparklySession):
 
         self.assertEqual(os_mock.environ, {
             'PYSPARK_PYTHON': sys.executable,
-            'PYSPARK_SUBMIT_ARGS': ' --jars file_a.jar,file_b.jar pyspark-shell',
+            'PYSPARK_SUBMIT_ARGS': '  --jars file_a.jar,file_b.jar pyspark-shell',
```

Comment:
if it's possible, let's normalise string so we don't need to test 2 vs 1 whitespace